obsei/analyzer/translation_analyzer.py

```diff
@@ -19,26 +25,55 @@ def __init__(self, **data: Any):
         self._pipeline = pipeline(
             "translation", model=model, tokenizer=tokenizer, device=self._device_id
         )
-        if hasattr(self._pipeline.model.config, 'max_position_embeddings'):
+        if hasattr(self._pipeline.model.config, "max_position_embeddings"):
             self._max_length = self._pipeline.model.config.max_position_embeddings
         else:
             self._max_length = 510
 
+    def _batchify(
+        self,
+        texts: List[str],
+        batch_size: int,
+        source_response_list: List[AnalyzerRequest],
+    ) -> Generator[Tuple[List[str], List[AnalyzerRequest]], None, None]:
+        for index in range(0, len(texts), batch_size):
+            yield (
+                texts[index : index + batch_size],
+                source_response_list[index : index + batch_size],
+            )
+
     def analyze_input(
-        self, source_response_list: List[AnalyzerRequest], **kwargs
+        self,
+        source_response_list: List[AnalyzerRequest],
+        analyzer_config: BaseAnalyzerConfig,
```

Comment:
Can you please run mypy, it will it seems type checking will fail some where.

Reply:
@lalitpagaria 
```bash
mypy obsei/analyzer/translation_analyzer.py --ignore-missing-imports
```

response:
Success: no issues found in 1 source file